{{cookiecutter.repo_name}}/package.json

```diff
@@ -93,26 +93,27 @@
     "dev": "gulp",
     "build": "gulp production",
     "lint:css": "stylelint \"{{cookiecutter.package_name}}/{assets/**,apps/*/assets/css}/*.css\"",
-    "lint:js": "eslint --ext .js,.vue {{cookiecutter.package_name}}/assets/js/**/*.js"
+    "lint:js": "eslint --ext .js,.vue {{cookiecutter.package_name}}/assets/js/{**/*.js,*.js}"
   },
   "lint-staged": {
     "linters": {
-      "*.js": [
+      "{{cookiecutter.package_name}}/assets/js/{**/*.js,*.js}": [
         "prettier-standard",
+        "eslint --fix",
         "git add"
       ],
       "*.vue": [
         "eslint --fix",
         "git add"
       ],
-      "**/!(definitions)/*.css": [
+      "{{cookiecutter.package_name}}/{assets/**,apps/*/assets/css}/*.css\"": [
```

Comment:
As this no longer excludes the definitions directory, does this depend on #213?

Reply:
Yep!